README.md

```diff
@@ -1,36 +1,47 @@
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+# AI Dashboard Generator
 
-## Getting Started
+AI Dashboard Generator is an open-source dashboard generator based on Next.js and ChatGPT. The tool allows users to feed existing tables or datasets, and from there, it understands the intent of the table and generates a visual representation of the data.
 
-First, run the development server:
+Internally, AI Dashboard Generator uses a custom CSV dataset and ChatGPT with a prompt model contextualized around JSON responses to give simple and useful visualizations. You can filter the parameters and export the dataviz directly into javascript code.
```

Comment:
I would rephrase this. Since the dashboard has those CSV datasets as examples so the user is not welcomed by a empty screen.
Also, same as before, we are not using the Chat.

Maybe something like "The Dashboard Generator uses a prompt engineered to make the AI reply using JSON to create simple and useful visualizaitons. We do not add any requirements for the titles or content, the AI does it all by using the data...."

Explaining a little bit that we are not actually hardcoding anything about the data,  just the sort of responses that we want.